### PR TITLE
got authenticated user

### PR DESCRIPTION
**Pull request by Rlhirmes103**
also setting up env for testing

### CODE REVIEW

**Comment by Rlhirmes103** [comment]
https://app.clubhouse.io/intello/story/1867/get-authenticated-user

File: zendesk/env/testing.env [hunk 1]
```diff
@@ -0,0 +1,3 @@
+ZENDESK_DOMAIN=intelloio
+ZENDESK_USERNAME=rachel@intello.io
+ZENDESK_PASSWORD=w96i2AD^V@&zUr6%$bc%W
```
Comment (sdalezman) [inline zendesk/env/testing.env]: @Rlhirmes103 you should never commit a password into source control. please change this asap.

we also shouldn't be introducing our own deps into external projects (like environator)

File: zendesk/user.go [hunk 2]
```diff
@@ -191,3 +191,9 @@ func (c *client) ShowComplianceDeletionStatuses(id int64) ([]ComplianceDeletionS
 		out)
 	return out.ComplianceDeletionStatuses, err
 }
+
+func (c *client) ShowCurrentlyAuthenticatedUser() (*User, error) {
```
Comment (sdalezman) [inline zendesk/user.go]: add a comment to an exported method

File: zendesk/zendesk.go [hunk 3]
```diff
@@ -160,7 +160,7 @@ func (c *client) request(method, endpoint string, headers map[string]string, bod
 	if err != nil {
 		return nil, err
 	}
-
+	//somehow this is the problem
```
Comment (sdalezman) [inline zendesk/zendesk.go]: ?

**Pull request closed without merging**